test/integration/service/service_test.go

```diff
@@ -113,27 +111,29 @@ var (
 )
 
 func init() {
-	check := func(err error) {
-		if err != nil {
-			panic(err)
-		}
-	}
-	ctx := context.Background()
+	// TODO: this required GLES support, which has since been removed.
```

Comment:
In general we remove commented out code. We can always keep the "Fix this" comment, and remove the code, since we can go back in history to find the code.

Reply:
I'll just nuke the body of this function then.